src/rules/utils.ts

```diff
@@ -341,6 +341,7 @@ export enum ModifierName {
 enum EqualityMatcher {
   toBe = 'toBe',
   toEqual = 'toEqual',
+  toStrictEqual = 'toStrictEqual',
```

Comment:
hah, only code change. awesome, thanks for verifying by adding some tests, though!

Reply:
Well, that's one question I can cross off my list :joy:

I had this pegged to ask about at some point, but never got around to it.
Cheers to all for putting a bow on this :)